packages/extension/src/view/devtools/pages/shared/cardsPanel.tsx

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies.
+ */
+import React from 'react';
+import { useSidebar } from '@google-psat/design-system';
+import classNames from 'classnames';
+
+export type Item = {
+  name: string;
+  icon: React.ComponentType<{ width: number; height: number }>;
+  sidebarKey: string;
+  colorClasses?: {
+    heading?: string;
+  };
+  description?: string;
+  buttons?: {
+    name: string;
+    sidebarKey: string;
+  }[];
+};
+
+type Props = {
+  pinnedItems?: Item[];
+  featuredItems: Item[];
+  onFeaturedButtonClick: (event: React.MouseEvent, sidebarKey: string) => void;
+};
+
+const CardsPanel = ({
+  pinnedItems,
+  featuredItems,
+  onFeaturedButtonClick,
+}: Props) => {
+  const navigateTo = useSidebar(({ actions }) => actions.updateSelectedItemKey);
+
+  return (
+    <div
+      data-testid="dashboard-content"
```

Comment:
If it is shared, will the test id be `dashboard-content` ?

Reply:
I think, instead of keeping the shared components in the `pages` folder, lets move it to `design-system` package.